bee-common/bee-common/src/logger/config.rs

```diff
@@ -57,7 +70,9 @@ pub struct LoggerOutputConfig {
     /// Name of an output file, or `stdout` for standard output.
     pub(crate) name: String,
     /// Log level of an output.
-    pub(crate) level: LevelFilter,
+    pub(crate) level_filter: LevelFilter,
+    /// Log filters of the output.
+    pub(crate) target_filters: Option<Vec<String>>,
```

Comment:
`Option` is needed for the builder to inform serde that the field is optional, but IMO in the built conf we should just have a vector, possibly empty. Option makes it more complex than it should be.